tests/test_spark.py

```diff
@@ -430,3 +433,75 @@ def test_task_stdout():
         assert stdout_file["size"] > 0, "stdout file should have content"
     finally:
         sdk_install.uninstall(utils.SPARK_PACKAGE_NAME, service_name)
+
+
+@pytest.mark.sanity
+def test_handling_wrong_request_to_spark_dispatcher():
+    service_name = utils.FOLDERED_SPARK_SERVICE_NAME
+    utils.require_spark(service_name=service_name)
+
+    json_data = sdk_cmd.run_cli("dcos task --json")
+    dispatcher_ip = json_data['statuses'][0]['container_status']['network_infos'][0]['ip_addresses'][0]['ip_address']
+    port = json_data['discovery']['ports'][0]['number']
+
+    headers = {
+        'Content-Type': 'application/json;charset=UTF-8',
+    }
+    data = {"action": "CreateSubmissionRequest",
+            "clientSparkVersion": "2.3.2",
+            "appResource": "noopTest.jar",
+            "mainClass": "noop.Test",
+            "appArgs": ["arg1", "arg2"],
+            "environmentVariables": {
+                "PATH": "/dev/null"
+            },
+
+            "sparkProperties": {
+                "spark.app.name": "TestDispatcher",
+
+            },
+
+            }
+    response = requests.post('http://{0}:{1}/submissions/create'.format(dispatcher_ip, port), headers=headers, data=data)
+
+    headers = {
```

Comment:
duplicate definition, variable with the same name and value is already defined in scope